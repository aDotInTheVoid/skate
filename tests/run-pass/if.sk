fn a(x: bool){
    if x {
        print "1"
    }
}

fn b(x: bool) {
    if x {} else {
        print 2
    }
}

fn c (x: bool) {
    if { return x } { print "no" } else { print "no" }
}

fn main() {
    a(true);
    print("a");
    a(false);
    print("b");
    b(true);
    print("a");
    b(false);
    print("b");
    c(true);
    c(false);
}