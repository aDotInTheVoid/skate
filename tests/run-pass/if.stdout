1
a
b
a
2
b
no
no
